satellite/satellitedb/migrate.go

```diff
@@ -441,6 +439,24 @@ func (db *DB) PostgresMigration() *migrate.Migration {
 					 ALTER TABLE users RENAME COLUMN last_name TO short_name;`,
 				},
 			},
+			{
+				Description: "drops interval seconds from storage_rollups, renames x_storage_rollups to x_storage_tallies, adds fields to bucket_storage_tallies",
```

Comment:
Describe at a higher level, this level of detail probably doesn't explain too much for the end-user.